XenAdmin/Diagnostics/Checks/DiskSpaceForBatchUpdatesCheck.cs

```diff
@@ -62,25 +59,33 @@ protected override Problem RunCheck()
             if (!Host.Connection.IsConnected)
                 throw new EndOfStreamException(Helpers.GetName(Host.Connection));
 
-            if (Helpers.CreamOrGreater(Host.Connection))
-            {
-                var action = new GetDiskSpaceRequirementsAction(Host, size, true, DiskSpaceRequirements.OperationTypes.automatedUpdates);
+            if (!Helpers.CreamOrGreater(Host.Connection))
+                return null;
+
+            var elyOrGreater = Helpers.ElyOrGreater(Host.Connection);
 
-                try
-                {
-                    action.RunExternal(action.Session);
-                }
-                catch
-                {
-                    log.WarnFormat("Could not get disk space requirements");
-                }
+            // check the disk space on host
+            var size = elyOrGreater

```

Comment:
Could we rename the variable to something like requiredDiskSize to make it clearer?

Reply:
👍